src/test/regress/expected/failure_copy_on_hash_1.out

```diff
@@ -0,0 +1,396 @@
+--
```

Comment:
I think the problem with normalizing this output was due to the multiline semantics.
I haven't looked into this now. But the rewrites are somewhat funky due to the following.

 - Later libpq versions sometimes have multiple occurances of `connection not open` all separated with by new lines
 - Errors that have different prefixes in older versions are the same in the newer function.
 
I agree that multiple output versions are sub-optimal. However, I think with this change the updates of multiple output files will become easier since we are streamlining the libpq version with the postgres version, eg. we can run them locally and have the same output as on CI.

If you have an idea on how to normalize we could go that route. However, I think we should normalize to the latest version (pg14 for now). But that output has a variable amount of `connection not open` lines, which I don't think we can realistically normalize correctly.

Reply:
Another rough idea: Can we only keep these latest outputs?

And, on failure_setup, we can maybe have `SELECT libpqVersion(); >= some_hard_coded_version` function in the tests -- with some comments, which fails if that is not greater or equals than the intended version? With that, even if the tests fails locally, the test points devs to install the right version.

I know, that also requires some level of maintenance, it'd fail when/if a new version changes outputs.

But, it'd prevent maintenance multiple files. (I'm assuming PG13 and PG 14 can have the same outputs)

Do you think would that work?